### PR TITLE
ng_ndp: enhance debug output

### PR DESCRIPTION
**Pull request by authmillenon**
Helped me a lot when doing #3385.

### CODE REVIEW

File: sys/net/network_layer/ng_ndp/ng_ndp.c [hunk 1]
```diff
@@ -969,6 +1019,13 @@ static void _set_state(ng_ipv6_nc_t *nc_entry, uint8_t state)
...
 
+#ifdef ENABLE_DEBUG
+        case NG_IPV6_NC_STATE_STALE:
+            DEBUG("STALE (go into DELAY on next packet)\n",
+                  ng_ipv6_addr_to_str(addr_str, &nc_entry->ipv6_addr,
+                                      sizeof(addr_str)));
+#endif
+
         default:
             break;
```
Comment (OlegHahm) [inline sys/net/network_layer/ng_ndp/ng_ndp.c]: `DEBUG("unkown\n"); ?

Otherwise the linebreak is missing.

Comment (authmillenon) [inline sys/net/network_layer/ng_ndp/ng_ndp.c]: Will add something in that spirit.

**Comment by OlegHahm** [comment]
ACK, please squash!

**Comment by authmillenon** [comment]
Squashed

**Pull request merged**